src/kvstore/raftex/Host.cpp

```diff
@@ -340,7 +340,7 @@ nebula::cpp2::ErrorCode Host::startSendSnapshot() {
             << ", lastLogId in wal = " << part_->wal()->lastLogId();
     sendingSnapshot_ = true;
     stats::StatsManager::addValue(kNumSendSnapshot);
-    part_->snapshot_->sendSnapshot(part_, addr_)
+    part_->snapshot_->sendSnapshot(part_, addr_, req->get_current_term())
```

Comment:
Why not use the `commitLogTerm` in `accessAllRowsInSnapshot`? If you use the term in `makeReq`, some other thing would go wrong, because it could be a latest term in the RaftPart as well?

Reply:
I realize one more problem, we can't simply use `commitLogTerm` I guess, because `verifyLeader` would check the term when follower receive the snapshot.